scripts/ci/docker-compose/integration-statsd.yml

```diff
@@ -19,15 +19,36 @@ version: "3.7"
 services:
   statsd-exporter:
     image: apache/airflow:airflow-statsd-exporter-2020.09.05-v0.17.0
+    container_name: "breeze-statsd-exporter"
```

Comment:
Naming the containers here breaks the existing breeze container naming convention which appears to be "let docker deal with it".  My Docker experience is pretty limited still, but AFAICT I needed a static docker container name here so i could reference it in the other files like [here](https://github.com/apache/airflow/pull/29449/files#diff-f03546a300946bd296ba407833ac290ee605c8ce586e484809513fe667c058f3R23).

As always, if there is a better way then please let me know.